red_october/red_october.py

```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
-# Copyright 2016 LasLabs Inc.
+# Copyright 2016-2017 LasLabs Inc.
 # License MIT (https://opensource.org/licenses/MIT).
 
+import json
```

Comment:
Why import the entire library instead of what you're using? :)

`from json import loads`